chimg/chroot.py

```diff
@@ -138,6 +138,10 @@ def _snap_install(self, name: str, channel: str, classic: bool = False, revision
         Install a single snap
         Use _snaps_install() to install all configured snaps
         """
+        # make sure the final target directories exist
+        Path(f"{self._ctx.chroot_path}/var/lib/snapd/seed/assertions").mkdir(parents=True, exist_ok=True)
+        Path(f"{self._ctx.chroot_path}/var/lib/snapd/seed/snaps").mkdir(parents=True, exist_ok=True)
```

Comment:
You can use a variable for `f"{self._ctx.chroot_path}/var/lib/snapd/seed/"`

Reply:
those paths are used all over the place. I think I'll extract those paths in another commit to have that unified.

Reply:
done now